docs/zh_cn/tutorials/0_config_.md

```diff
@@ -1,3 +1,263 @@
 # 教程0: 如何使用 Configs
 
-欢迎有兴趣的朋友一起翻译 MMFlow 文档。如有兴趣，请在 [MMFlow issue]() 提 issue 确定翻译的文档。
+我们的配置文件 (configs) 中支持了模块化和继承设计，这便于进行各种实验。如果需要检查配置文件，可以通过运行 `python tools/misc/print_config.py /PATH/TO/CONFIG` 来查看完整的配置文件。
+
+## 配置文件结构
+
+在目录 `config/_base_` 下有四种基本模块类型，即数据集 (datasets) 、模型 (models) 、训练计划 (schedules) 以及默认运行配置 (default_runtime)。很多模型可以很容易地参考其中一种方法 (如 PWC-Net )来构建。这些由 `_base_` 中的模块构成的配置文件被称为 *原始配置 (primitive configs)*。
+
+对于同一文件夹下的所有配置，建议只有**一个**原始配置。所有其他配置都应该从原始配置继承。这样，最大继承级别 (inheritance level) 为 3。
+
+简单来说，我们建议贡献者去继承现有模型的配置文件。例如，如果在 PWC-Net 的基础上做了一些改动，可以首先通过在配置文件中指定原始配置 `_base_ = ../pwcnet/pwcnet_slong_8x1_flyingchairs_384x448.py` 来继承基本的 PWC-Net 结构，然后再根据需要修改配置文件中的指定字段 (fields)。
+
+如果需要搭建不能与现有模型共享任何结构的全新的模型，您可以在 `configs` 下创建一个新文件夹 `xxx`。
+
+您也可以参考 [mmcv](https://mmcv.readthedocs.io/en/latest/understand_mmcv/config.html) 文档中的更多细节。
+
+## 配置文件命名规则
+
+我们按照下面的风格来命名配置文件。建议贡献者使用相同的风格。
+
+```text
+{model}_{schedule}_[gpu x batch_per_gpu]_{training datasets}_[input_size].py
+```
+
+`{xxx}` 表示必填字段，而 `[yyy]` 表示可选字段。
+
+- `{model}`: 模型类型，如 `pwcnet`, `flownets` 等等。
+- `{schedule}`: 训练计划。按照 FlowNet2 中的约定，我们使用 `slong`、 `sfine` 和 `sshort`，或者像 `150k` 表示150k(iterations) 这样指定迭代次数。
+- `[gpu x batch_per_gpu]`: GPU 数量以及每个 GPU上分配的样本数， 如 `8x1`。
+- `{training datasets}`: 训练数据集，如 `flyingchairs`， `flyingthings3d_subset` 或 `flyingthings3d`。
+- `[input_size]`: 训练时图片大小。
+
+## 配置文件结构
+
+为了帮助用户对完整的配置和 MMFlow 中的模块有一个基本的了解，我们以在 `flyingchairs` 上使用 `slong` 训练的 PWC-Net 的配置为例进行简单的讲解。有关每个模块的更详细的用法和相应的替代方案，请参阅 API 文档和 [MMDetection 教程](https://github.com/open-mmlab/mmdetection/blob/master/docs/tutorials/config.md)。
+
+```python
+_base_ = [
+    '../_base_/models/pwcnet.py', '../_base_/datasets/flyingchairs_384x448.py',
+    '../_base_/schedules/schedule_s_long.py', '../_base_/default_runtime.py'
+]# base config file which we build new config file on.
```

Comment:
might translate comments in code in Chinese

Reply:
Ok.